MastodonSDK/Sources/MastodonUI/View/Content/MediaBadgesContainer.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MediaBadgesContainer.swift
+//
+//  Created by Jed Fox on 2022-12-20.
+//
+
+import SwiftUI
+
+struct MediaBadgesContainer: View {
+    var altDescription: String?
+    var isGIF = false
+    var videoDuration: TimeInterval?
+    
+    @State private var showingAlt = false
+    @State private var space = AnyHashable(UUID())
+
+    // Date.ComponentsFormatStyle does not allow force-enabling minutes unit
+    static let formatter: DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
+        formatter.allowedUnits = [.minute, .second]
+        formatter.unitsStyle = .positional
+        formatter.zeroFormattingBehavior = []
+        formatter.formattingContext = .standalone
+        return formatter
+    }()
+
+    var body: some View {
+        GeometryReader { geom in
+            HStack(alignment: .bottom, spacing: 2) {
+                if let altDescription {
+                    ExpandableMediaBadge("ALT", isExpanded: $showingAlt, in: (geom.size, space)) {
+                        Text(altDescription)
+                            .frame(maxHeight: geom.size.height - 16)
+                            .fixedSize(horizontal: false, vertical: true)
+                    }
+                }
+                if isGIF {
+                    MediaBadge("GIF")
+                }
+                if let videoDuration, let format = Self.formatter.string(from: videoDuration) {
+                    MediaBadge(format)
```

Comment:
Above you use `.monospacedDigit()` in the preview. Here it's missing. Is there a reason for that?

Reply:
🙈